streamwebs_frontend/streamwebs/views.py

```diff
@@ -104,7 +63,6 @@ def user_login(request):
     if request.method == 'POST':
         username = request.POST['username']
         password = request.POST['password']
-
         user = authenticate(username=username, password=password)
 
         if user:
```

Comment:
Forgot there was more garbage. This time it's line 72 if you're interested in getting rid of it now.